posthog/hogql/test/test_printer.py

```diff
@@ -303,6 +303,8 @@ def test_comparisons(self):
         self.assertEqual(self._expr("event not ilike 'E'", context), "not(ilike(events.event, %(hogql_val_9)s))")
         self.assertEqual(self._expr("event in 'E'", context), "in(events.event, %(hogql_val_10)s)")
         self.assertEqual(self._expr("event not in 'E'", context), "not(in(events.event, %(hogql_val_11)s))")
+        self.assertEqual(self._expr("event =~ 'E'", context), "match(events.event, %(hogql_val_12)s)")
```

Comment:
Why the `=`? This seems inspired by Postgres, which is AFAIK the only SQL flavor with symbolic operator for regexes, where this would be just `~`

Reply:
Hmm... Good point. I'm used to `=~` from other languages (perl, ruby). We could support both, for even more consistency between `==`, `=` (same) and `!=`